build.sbt

```diff
@@ -1,11 +1,14 @@
 import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
 
+val adjectiveVersion = "0.5.0"
```

Comment:
I'd remove version, as the build starts without it, see [this](https://app.circleci.com/pipelines/github/vivri/Adjective/10/workflows/1a666d82-a74f-46ee-8597-de8136c80a90/jobs/41).

Reply:
Also, having `version` would disrupt the release plugin, as discussed over chat.

Reply:
Ok, I removed the version from sbt!